spec/support/capybara.rb

```diff
@@ -1,5 +1,17 @@
+require "selenium/webdriver"
+
 Capybara.register_driver :chrome do |app|
   Capybara::Selenium::Driver.new(app, browser: :chrome)
 end
 
-Capybara.javascript_driver = :chrome
+Capybara.register_driver :headless_chrome do |app|
+  capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
+    chromeOptions: { args: %w(headless disable-gpu) }
```

Comment:
I meant to remove that from the options here, but I'm happy enough to leave it in. I don't want to use my GPU in tests.